### PR TITLE
Adding UDP Support

### PR DESCRIPTION
**Pull request by AlecHFerguson**
We wanted to add UDP support to Good Influx.  In order to do this, I created a wrapper function around an HTTP reporter and a UDP reporter.

The diff is a little wonky since I moved `index.js` to `good-influx-http.js`; then created an `index.js` that is just a wrapper function.

### CODE REVIEW

**Comment by estk** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -17,9 +20,9 @@ const Hapi = require('hapi');
 const server = new Hapi.Server();
 server.connection();
 
-const options = {
+const goodOptions = {
```
Comment (wookiehangover) [inline README.md]: I'd say this is probably an unnecessary — or at very least unrelated  — change for this particular PR

File: lib/good-influx-http.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+class GoodInfluxHttp extends GoodHttp {
+    constructor(endpoint, config) {
+        super(endpoint, config);
+        this._settings.schema = 'good-influx';
+        this.config = Hoek.applyToDefaults(internals.defaults, config)
+    }
+    _sendMessages(callback) {
+
```
Comment (wookiehangover) [inline lib/good-influx-http.js]: nit: extra whitespace

File: README.md (hunk 1 above)
Comment (AlecHFerguson) [inline README.md]: True.  I do think it makes it easier for users new to the Good ecosystem (like myself, recently) to understand how good configuration works.  So I would prefer to keep it in.

What does the team think?  Any objections to renaming this variable in the usage example in the README?

**Pull request merged**